GAME_DESIGN_PRINCIPLES.md

```diff
@@ -0,0 +1,100 @@
+
+### Fuel the Player's Curiosity
+The world is filled with secrets, and we want the player to find these secrets on their own without any outside help or guide.
+
+### Problems have Multiple Solutions
+Problems faced in a particular age of progression should have multiple solutions, instead of just one solution for everything.
+For example, in the "Pre-Caves Age," the player will not have access to coal, as coal spawns low in the world and is shrouded in darkness. There's many solutions to this, however:
+- Find a dim, above ground light source.
+- Search for a cave with exposed coal.
+- Wander into the darkness to find a cave that's bright enough to mine in.
+- Find an above-ground structure that can grant the player coal.
+- Dig straight down until you find coal.
+
+Of course, this problem is completely solved as soon as the player gets coal, as they can now explore caves to find more coal with their newfound torches, but it's important that the player has these options in the first place so they can have more freedom. Not one option is better or easier than the other, and if one turns out to be, then it should be allowed to be adjusted to encourage other methods.
+
+### Explain when Needed
+If something progress-related has no obvious explanation, the player will have to rely on a wiki to find out how to progress. One example is the bellows; players won't know to place it next to a furnace and jump on it repeatedly. The issue can be fixed by adding a tooltip that tells them how it works, while leaving out details they can intuitively find out themselves.
+
+### Micro Moments
+These are tiny things the player does in between larger events; examples include:
+- Travelling
+- Parkour
+- Mining
+- Building
+- Crafting
+- Managing inventory
+- Fighting
+
+These moments are extremely important as they let the player mentally rest, so making sure they're as satisfying and consistent as possible is a must.
+
+# Balancing
+When balancing the game, keep in mind how players might interact with the world, the wildlife, and each other.
+
+### 2OP4ME
+At no point should the player be extremely hard to kill. Armor, tools, accessories, and buffs should be designed around aiding the player in skill-based encounters, not making them win regardless of skill.
+
+### Trade-offs
+If the player is given a powerful ability, then it should have an appropriate downside that forces them to use said ability or interact with other mechanics.
+
+**Examples:**
+- An accessory that gives the player a double-jump, but increases their fall damage.
+- An accessory that heals the player, but takes away energy or some other resource.
+- Armor that grants a lot of defense, but makes the player slow.
+- A golden pickaxe is strong, but uses rare resources and is slow to swing.
+
+### No Unbreakable Tools
+There should be no way to make tool last indefinitely.
```

Comment:
You need to explain why. In Minecraft tool is an investment, one has to invest resources and time to make the tool great and then they don't want it to break. In Cubyz, a tool is just it - a tool. You make it, you use it, you throw it away, there is no long upgrade path for single tool, progression is horizontal, with tone of different combinations and materials, but after you make that tool it becomes a tool and there is no more investment to make. If it breaks, you just go to a cave to find more materials, they are waiting there, you don't have to grind XP for random enchants, in Cubyz all is deterministic.